spec/models/evaluator_invitation_spec.rb

```diff
@@ -2,35 +2,37 @@
 
 RSpec.describe EvaluatorInvitation, type: :model do
   let(:challenge) { create(:challenge) }
-  let(:phase) { create(:phase, challenge: challenge) }
+  let(:phase) { create(:phase, challenge:) }
 
   it "can be created with valid attributes" do
-    invitation = build(:evaluator_invitation, challenge: challenge, phase: phase)
+    invitation = build(:evaluator_invitation, challenge:, phase:)
     expect(invitation).to be_valid
-    expect { invitation.save! }.to change(EvaluatorInvitation, :count).by(1)
+    expect { invitation.save! }.to change { described_class.count }.by(1)
   end
 
   it "can be destroyed" do
-    invitation = create(:evaluator_invitation, challenge: challenge, phase: phase)
-    expect { invitation.destroy }.to change(EvaluatorInvitation, :count).by(-1)
+    invitation = create(:evaluator_invitation, challenge:, phase:)
+    expect { invitation.destroy }.to change { described_class.count }.by(-1)
   end
 
   it "validates uniqueness of email within challenge and phase" do
-    create(:evaluator_invitation, challenge: challenge, phase: phase, email: "test@example.com")
-    duplicate_invitation = build(:evaluator_invitation, challenge: challenge, phase: phase, email: "test@example.com")
+    create(:evaluator_invitation, challenge:, phase:, email: "test@example.com")
+    duplicate_invitation = build(:evaluator_invitation, challenge:, phase:, email: "test@example.com")
     expect(duplicate_invitation).not_to be_valid
     expect(duplicate_invitation.errors[:email]).to include("has already been taken")
   end
 
   it "allows the same email for different challenges or phases" do
-    create(:evaluator_invitation, challenge: challenge, phase: phase, email: "test@example.com")
+    create(:evaluator_invitation, challenge:, phase:, email: "test@example.com")
     different_challenge = create(:challenge)
     different_phase = create(:phase, challenge: different_challenge)
 
-    invitation_different_challenge = build(:evaluator_invitation, challenge: different_challenge, phase: phase, email: "test@example.com")
+    invitation_different_challenge = build(:evaluator_invitation, challenge: different_challenge, phase:,
+                                                                  email: "test@example.com")
```

Comment:
is this line spacing from the linter? looks kinda weird, maybe the other key/value pairs should be on separate lines too?